src/components/WalletSelect.vue

```diff
@@ -23,12 +23,22 @@ export default {
     setWallet(wallet: Wallet) {
       this.paymentsStore.setWallet(wallet)
     },
+    chooseValr() {
+      this.valrSelected = true
+    },
+    setValr(currency: string) {
+      this.paymentsStore.setPaymentCurrency(currency)
+      this.paymentsStore.setWallet(Wallet.wallets['valr'])
+    },
   },
   data() {
     return {
-        Wallet: Wallet
+        valrSelected: false,
+        Wallet: Wallet,
+        valrCurrencies: ["BTC","AVAX", "BNB", "ETH", "EURC", "PYUSD", "SHIB", "SOL", "USDC", "USDT", "XRP"],
```

Comment:
How did you pick the currencies to list and the order?

Reply:
I see it's the same list as in CryptoQR

Reply:
For Luno:

|payment_currency|total_zar|count|
|----------------|---------|-----|
|BTC|2452611|5865|
|ETH|320642|855|
|XRP|305899|1335|
|SOL|88271|340|
|USDC|64670|149|
|LTC|51442|116|
|ADA|41865|178|
|USDT|36600|93|
|BCH|16934|66|
|XLM|15587|73|
|AVAX|12899|51|
|MATIC|12757|91|
|LINK|12465|40|
|UNI|11584|43|
|CRV|7961|35|
|DOT|6957|47|
|TRX|4605|46|
|ALGO|3647|37|
|ATOM|3333|22|
|MKR|3063|15|
|AAVE|2116|12|
|SNX|459|8|
|GRT|408|5|
|FTM|195|2|
|NEAR|109|1|

Reply:
yeah, I can add ZAR. I pulled all the VALR currencies and filtered for ones that have a ZAR market, which is what CQR does. In this case it just made more sense to hard-code it than to pull it dynamically. easy enough to fix later if we ever need to